packages/kitsu-core/src/deepEqual/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Compare two objects equality 
+ *
+ * @name isDeepEqual
```

Comment:
Name is not required as it is inferred from the function name by the documentation tooling